### DISCUSSION TITLE
What if all of nodes are die?

### DISCUSSION THREAD

**Discussion by nowjean** [ConsenSys/quorum#1644] [Q&A]
Hi
I'm running besu on Kubernetes.
I wondering what if all of nodes are die and then recreated?

Follwing the Besu docs, (https://besu.hyperledger.org/en/stable/private-networks/how-to/configure/consensus/ibft/)

<q>_Configure your network to ensure you never lose more than ⅓ of your validators. If more than ⅓ of validators stop participating, new blocks are no longer created, and the network stalls. It may take significant time to recover once nodes are restarted._</q>

Then, If all of nodes are recreated, block chain network working again?

**Comment by baptiste-b-pegasys** [comment]
Nodes should be nicely killed so that all data are properly written on disk.

If so, yes it should boot again.

If one node is out of sync, synchronization should happen.

The network need the consensus quorum to move along to the next block. 

But the catch is if one node has a more recent block is not part of the network when building a new block, this can lead to a fork.

It's possible to reset forked node to the common block (block before fork) with debug set head.
